src/libsyntax/ast.rs

```diff
@@ -220,8 +222,9 @@ impl Path {
     }
 }
 
-/// A segment of a path: an identifier, an optional lifetime, and a set of
-/// types.
+/// A segment of a path: an identifier, an optional lifetime, and a set of types,
+///
+/// e.g. `std`, `String` or `Box<T>`
```

Comment:
Oh, I now see you use this style throughout the file, which makes sense. I'd probably just change it to start with a capital letter ("E.g.") or write the long for ("For example").